src/ai_client.c

```diff
@@ -3,6 +3,8 @@
  *
  * Copyright (C) 2009-2011 Keldon Jones
  *
+ * Source file modified by J.-R. Reinhard, October 2016.
```

Comment:
Well, there is a slight deviation in the communication protocol during XI games, where the value of a bonus against Xenos might be exchanged, cf server.c, line 1975. Since it only occurs in XI games, the value is only exchanged during these kind of games and thus does not break the protocol version, unless both parties support XI. With this in mind, do you still support the definition of COMM_VERSION set to 0.9.4?